src/app/views/censusdata/create2.html

```diff
@@ -0,0 +1,73 @@
+<div class="container">
+    <div ng-include="'/app/views/censusdata/progressBar.html'"></div>
+    {{censusForm.step = 2}}
+    <form class="form-horizontal" role="form"> 
+        <div class="form-group panel panel-primary">
+            <div class="panel-heading">Waterbird Survey<span style="float:right;">Report violations to 1-800-792-4263</span></div>
+            <div class="panel-body">
+            <!--<div>{{ 4/2 }}</div>-->
+            <div class="form-group">   
+                <div class="col-md-12">        
+                    <!--<table class="table table-condensed">-->
+                    <table class="table table-striped">
+                        <thead>
+                        <tr class="info">
+                            <th>
+                            </th>
+                            <th>No. of Adults</th>
+                            <th>No. of Juvenile</th>
+                            <th>Primary Activity</th>
+                            <th>Secondary Activity</th>
+                            <th>Location/ Habitat Type</th>
+                            <th>% Feeding Success Rate</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                       <!--TODO: Once birdSpicies/GetBySurveyId is there limitTo filter will be removed.-->
+                        <tr ng-repeat="(i, bird) in birdSpeciesList  | limitTo:10  track by $index">
+                            <td>{{bird.commonName}}
```

Comment:
Angularjs creates a property in $scope when textboxes changes. And therefore I wanted to give birdId as default value to hidden field hoping that it will be automatically added to the scope but that did not work!